src/Me/Modals/EditMentorDetails.js

```diff
@@ -174,22 +186,29 @@ function EditMentorDetails({
                 const isDisabled =
                   filledChannel.length >= 3 && !(propData && propData.id);
                 return (
-                  <ExtendedFormField key={option.value} label={option.label} helpText={option.helpText}>
-                    <Input
-                      aria-labelledby={option.value}
-                      type="text"
-                      name={`${fieldName}[${option.value}]`}
-                      value={propData ? propData.id : ''}
-                      onChange={e => {
-                        handleKeyValueChange(
-                          fieldName,
-                          option.value,
-                          e.target.value
-                        );
-                      }}
-                      disabled={isDisabled}
-                      placeholder={option.placeholder}
-                    />
+                  <ExtendedFormField
+                    key={option.value}
+                    label={option.label}
+                    helpText={option.helpText}
+                  >
+                    <div>
+                      <span>{option.prefix}</span>
+                      <Input
+                        aria-labelledby={option.value}
+                        type="text"
+                        name={`${fieldName}[${option.value}]`}
+                        value={propData ? propData.id : ''}
```

Comment:
it's perfectly valid but its also an opportunity for optional chaining and Nullish coalescing operator 😲
`propData?.id ?? ''`

Reply:
I'll leave it to your PR :)